magento/2.4/docker-entrypoint.sh

```diff
@@ -2,23 +2,6 @@
 
 [ "$DEBUG" = "true" ] && set -x
 
-PERSISTENT_FOLDER_LIST=('var/composer_home' 'var/log' 'var/export' 'var/report' 'var/import' 'var/import_history' 'var/session' 'var/importexport' 'var/backups' 'var/tmp' 'pub/cache' 'pub/sitemap' 'pub/media')
```

Comment:
I thought that those folders will be created in the Dockerfile instead of the entrypoint, but I don't see it 🤔

Reply:
Those folders are created from the entrypoint.sh in the package ".artifakt" given to customer. That's here that happens!